proto/spire/common/common.proto

```diff
@@ -83,6 +83,8 @@ message RegistrationEntry {
     repeated string dns_names = 10;
     /** Revision number is bumped every time the entry is updated */
     int64 revision_number = 11;
+    /** Determines if the issued identity is exportable to a store */
+    bool store_svid = 12;
```

Comment:
changes on common protos are done to be used for agent but no changes is applied to server side, changes to server will go in a separated PR